### PR TITLE
Update dynamic asg docs with capi v2 endpoint info

### PR DESCRIPTION
**Pull request by geofffranks**

### CODE REVIEW

File: docs/dynamic-asg-overview.md [hunk 1]
```diff
@@ -113,3 +114,11 @@ Dynamic ASGs is a linux-only feature. Currently there is no vxlan-policy-agent
 equivalent for windows cells. If this is a feature you are interested in, please
 open an issue as a feature request.
 
+### 8. Why aren't my security-group updates taking a very long time to be detected and synced through to policy-server?
+
+Because only `/v3/security_groups` endpoints update Cloud Controller's internal timestamp for when ASG info has changed, any
+ASG create/update/delete activity initiated using the `/v2/security_groups` will go unnoticed by `policy-server-asg-syncer`.
+However, upon the next ASG modification using the CF API v3 endpoints, a full sync will occur and the changes made using the
+CF API v2 endpoints will be pulled in at this time. As the CF API v2 has been dprecated for some
```
Comment (ebroberson) [inline docs/dynamic-asg-overview.md]: Typo on depr**e**cated

File: docs/dynamic-asg-overview.md [hunk 2]
```diff
@@ -113,3 +114,11 @@ Dynamic ASGs is a linux-only feature. Currently there is no vxlan-policy-agent
 equivalent for windows cells. If this is a feature you are interested in, please
 open an issue as a feature request.
 
+### 8. Why aren't my security-group updates taking a very long time to be detected and synced through to policy-server?
```
Comment (ebroberson) [inline docs/dynamic-asg-overview.md]: Is this supposed to be "aren't"?  It seems like it should be "why are ... taking a very long time".

Comment (geofffranks) [reply docs/dynamic-asg-overview.md]: uh huh.

File: docs/dynamic-asg-overview.md (hunk 1 above)
Comment (geofffranks) [reply docs/dynamic-asg-overview.md]: TY

**Comment by geofffranks** [comment]
@ebroberson - Fixed both items

**Review by ebroberson — APPROVED** [review]
LGTM

**Pull request merged**